design/KruizeConfigurationProcedure.md

```diff
@@ -48,6 +48,14 @@ data:
         "c3p0maxstatements": 50,
         "hbm2ddlauto": "update",
         "showsql": "true"
+      },
+      "cloudwatch": {
+        "accessKeyId": "",
```

Comment:
Update the environment variables section if these can be configured through env variables.

Reply:
@kusumachalasani - Are the env variables same as that in the manifest to pass the cloudwatch keys?